dev_tools/devspace.yaml

```diff
@@ -55,7 +55,7 @@ dev:
     labelSelector:
       control-plane: odh-model-controller
     # Replace the container image with this dev-optimized image (allows to skip image building during development)
-    devImage: quay.io/rh-ee-allausas/golang:1.22-odh-devspace-debug
+    devImage: quay.io/rh-ee-allausas/golang:1.24.2-odh-devspace
```

Comment:
Does this image exist?
@andresllh may we use the go1.24 from RH registry? Or, is there any customization on this image?